### PR TITLE
Implemented Standalone App

### PR DESCRIPTION
**Pull request by jacobmillsfl**
- Standalone app now fully functional
- Underlying CommandUtil routines implemented

### CODE REVIEW

File: src/utils/command_interface.py [hunk 1]
```diff
@@ -41,41 +47,133 @@ class CommandInterface:
...
+    def command_claim(self, discord_id: int, leetcode_id: str) -> str:
         """
         Associates a leetcode_id with a discord_id in the Leetcode_User table.
         Local only, no API required.
         Database required.
         """
-        pass
+        message = ""
+        create_user = False
+        claimed_user = self.database.table_leetcodeuser_load_by_discord_id(leetcode_id)
```
Comment (JakeTompkins) [inline src/utils/command_interface.py]: Looks like you mean to use the by_leetcode_id method

File: src/utils/command_interface.py [hunk 2]
```diff
@@ -41,41 +48,135 @@ class CommandInterface:
...
         """
         Determines the current weekly challenge.
         Local only, no API required.
         Database required.
         """
-        pass
+        latest = self.database.table_weeklychallenge_getlatest()
+        if latest:
+            start_date = datetime.fromtimestamp(latest["date"])
+            result = f"* * * CHALLANGE {latest['id']} | {start_date.date()} * * *\n\n"
```
Comment (jacobmillsfl) [inline src/utils/command_interface.py]: Typo in CHALLENGE

Comment (JakeTompkins) [inline src/utils/command_interface.py]:
```suggestion
            result = f"* * * CHALLENGE {latest['id']} | {start_date.date()} * * *\n\n"
```

File: src/utils/command_interface.py [hunk 3]
```diff
@@ -41,41 +48,135 @@ class CommandInterface:
...
+            else:
+                result += f"User {leetcode_user_id}'s Weekly Challenge status:\n"
+                questions = self.database.table_weeklyquestion_load_by_challenge_id(challenge["id"])
+                for question in questions:
+                    complete = LeetcodeUtil.check_challenge_completion(leetcode_user_id
+                                                                       , question["title_slug"])
+                    result += f"\t{'Complete' if complete else 'Incomplete'}"\
+                        f"\t-\t{question['title']}\n"
+        else:
+            result += "Leetcode user not found. Claim your user idea with"
```
Comment (JakeTompkins) [inline src/utils/command_interface.py:155]:
```suggestion
            result += "Leetcode user not found. Claim your user id with"
```

**Review by JakeTompkins — CHANGES_REQUESTED** [review]

**Review by lcollins11 — APPROVED** [review]

**Review by JakeTompkins — APPROVED** [review]

**Pull request merged**